config/dev.exs

```diff
@@ -1,33 +1,12 @@
 import Config
 
-# Chain
-config :aecore, network_id: System.get_env("NETWORK_ID", "ae_mainnet")
-
 # Telemetry
 config :ae_mdw, :enable_livedashboard, true
 
 config :ae_mdw, TelemetryMetricsStatsd,
   host: "localhost",
   port: 8125
 
-# Endpoints
-protocol_opts = [max_request_line_length: 1_024, max_skip_body_length: 1_024]
-
-config :ae_mdw, AeMdwWeb.Endpoint,
```

Comment:
Some of these are needed in dev, used differently from prod. I see there is a boolean flag on `debug_errors` in the `runtime.exs`. Could we add the same for `live_view`? Or just leave the endpoint declaration here in `dev.exs`.

Reply:
Back to dev too